### DISCUSSION TITLE
CSS op styles the editor

### DISCUSSION THREAD

**Discussion by tgd-git** [cables-gl/cables_docs#264] [! Welcome - readme first!]
Hey not sure how this works but i just styled the editor window with the css op. see the white border on everything. This is not supposed to happen right? ;) it is dev.cables.gl btw.

![image](https://user-images.githubusercontent.com/2562637/172349559-2946bade-7adb-412b-adff-bccf1d850ea4.png)

**Comment by pandrr** [comment]
this is supposed to happen

**Reply by tgd-git** [reply]
really? can not reproduce it though

**Reply by steam0r** [reply]
some css rules in the editor-stylesheet are more specific than the ones in your op, so they "win"
